app/api/v1/endpoints/chat_session_endpoints.py

```
@@ -45,166 +45,186 @@
+            .execute()
+        
+        if response.data:
+            return response.data[0]
+        else:
+            raise Exception("Failed to create message")
+            
+    except HTTPException:
+        raise
+    except Exception as e:
+        logger.error(f"Error creating chat message: {e}")
+        raise HTTPException(status_code=500, detail=str(e))
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)